src/java/org/apache/cassandra/db/virtual/SystemViewsKeyspace.java

```diff
@@ -50,6 +53,8 @@ private SystemViewsKeyspace()
                     .add(new QueriesTable(VIRTUAL_VIEWS))
                     .add(new LogMessagesTable(VIRTUAL_VIEWS))
                     .addAll(LocalRepairTables.getAll(VIRTUAL_VIEWS))
+                    .add(new AnalyzerView(VIRTUAL_VIEWS))
+                    .add(new IndexesSystemView(VIRTUAL_VIEWS))
```

Comment:
I wonder if we could do something like what we did w/ `LocalRepairTables` and have a container class, perhaps `org.apache.cassandra.db.virtual.StorageAttachedIndexTables`, that would manage the tables exposed here via a `getAll()` method, etc.